### PR TITLE
134-SQL-Queries

### PR DESCRIPTION
**Pull request by MilRoad**
Issue #134

### CODE REVIEW

File: HW4/Milena Laistseva/SQLqueries/dbo/Stored Procedures/133CustomersCountriesWithoutBetween.sql [hunk 1]
```diff
@@ -0,0 +1,6 @@
+﻿CREATE PROCEDURE [dbo].[133CustomersCountriesWithoutBetween]
+AS
+	SELECT CustomerID, Country
+FROM Customers
+WHERE Country >='B' AND Country < 'H'
```
Comment (vladimirchr) [inline HW4/Milena Laistseva/SQLqueries/dbo/Stored Procedures/133CustomersCountriesWithoutBetween.sql]: What an interesting solution!)

File: HW4/Milena Laistseva/SQLqueries/dbo/Stored Procedures/222EmployeeAmountOfOrders.sql [hunk 2]
```diff
@@ -0,0 +1,12 @@
+﻿CREATE PROCEDURE [dbo].[222EmployeeAmountOfOrders]
+	@param1 int = 0,
+	@param2 int
```
Comment (vladimirchr) [inline HW4/Milena Laistseva/SQLqueries/dbo/Stored Procedures/222EmployeeAmountOfOrders.sql]: Did you forget to delete this?)

File: HW4/Milena Laistseva/SQLqueries/dbo/Stored Procedures/112UndeliveriedOrders_1.sql [hunk 3]
```diff
@@ -0,0 +1,9 @@
+﻿CREATE PROCEDURE [dbo].[112UndeliveriedOrders]
```
Comment (vladimirchr) [inline HW4/Milena Laistseva/SQLqueries/dbo/Stored Procedures/112UndeliveriedOrders_1.sql]: We have two files with same content.

File: HW4/Milena Laistseva/SQLqueries/dbo/Stored Procedures/113DeliveriedByDateOrUndeliveriedOrders.sql [hunk 4]
```diff
@@ -0,0 +1,10 @@
+﻿CREATE PROCEDURE [dbo].[113DeliveriedByDateOrUndeliveriedOrders]
```
Comment (vladimirchr) [inline HW4/Milena Laistseva/SQLqueries/dbo/Stored Procedures/113DeliveriedByDateOrUndeliveriedOrders.sql]: We have two procedures with different name but same content)

File: HW4/Milena Laistseva/SQLqueries/dbo/Stored Procedures/224CustomerAndEmployeeSameCity.sql [hunk 5]
```diff
@@ -0,0 +1,7 @@
+﻿CREATE PROCEDURE [dbo].[224CustomerAndEmployeeSameCity]
+AS
+	SELECT
+CONCAT (Employees.LastName, ' ', Employees.FirstName) AS Seller,
+Customers.ContactName AS Customer
+FROM Employees, Customers
+WHERE Employees.City = Customers.City
```
Comment (alch-0) [inline HW4/Milena Laistseva/SQLqueries/dbo/Stored Procedures/224CustomerAndEmployeeSameCity.sql]: It appears that according to the assignment the query must return one column with the names of both sellers and buyers. It is not indicated directly, but the assignment also says not to use JOIN.
While this query does not use the JOIN keyword explicitly, it is nonetheless an implicit inner join.

File: HW4/Milena Laistseva/SQLqueries/dbo/Stored Procedures/222EmployeeAmountOfOrders.sql (hunk 2 above)
Comment (MilRoad) [reply HW4/Milena Laistseva/SQLqueries/dbo/Stored Procedures/222EmployeeAmountOfOrders.sql]: Yeah :D

File: HW4/Milena Laistseva/SQLqueries/dbo/Stored Procedures/133CustomersCountriesWithoutBetween.sql (hunk 1 above)
Comment (MilRoad) [reply HW4/Milena Laistseva/SQLqueries/dbo/Stored Procedures/133CustomersCountriesWithoutBetween.sql]: Thanks :)

File: HW4/Milena Laistseva/SQLqueries/dbo/Stored Procedures/224CustomerAndEmployeeSameCity.sql (hunk 5 above)
Comment (MilRoad) [reply HW4/Milena Laistseva/SQLqueries/dbo/Stored Procedures/224CustomerAndEmployeeSameCity.sql]: Okay, I've changed it

**Pull request merged**